validators/Aadhar_IN.js

```diff
@@ -0,0 +1,14 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
```

Comment:
1. Aadhar card number can be in string or number
2. Aadhar length needs to be checked
3. `x` should be overwritten with to use `.length` and `.match` in future execution